src/components/settings/SettingsApiCredentialsTab/CopyButton.jsx

```diff
@@ -12,6 +12,8 @@ const CopyButton = ({ data }) => {
 
   const handleCopyLink = async () => {
     try {
+      const newData = data;
+      ['user', 'id'].forEach(prop => delete newData[prop]);
```

Comment:
Do we need to make a newData variable here?  Because you're just making a reference copy, this statement is equivalent to just deleting from the original data variable.

Reply:
We just had some eslint things about making direct changes to a parameter.